oxidation/libparsec/crates/testbed/src/coolorg.rs

```diff
@@ -151,21 +158,21 @@ pub(crate) fn generate() -> TestbedTemplate {
     // Local devices
     let device_files = vec![
         TestbedDeviceFileData::new(
-            &alice1_id,
+            alice1_id,
             "P@ssw0rd.".to_owned(),
             SecretKey::from(hex!(
                 "323614fc6bd2d300f42d6731059f542f89534cdca11b2cb13d5a9a5a6b19b6ac"
             )),
         ),
```

Comment:
Could you create a macro for that ?

Reply:
Why ?

Reply:
To factorize code, to have less text to read :shrug:

Reply:
8 * 3 = 24 lines into 4 * 3 + 10 = 22 lines + a macro only for that ?

Reply:
So no macro